### PR TITLE
added link to place where community can comment on data sources

### PR DESCRIPTION
**Pull request by lhawthorn**
This will eventually need to be more streamlined than an open issue, but for now an open issue suffices.

Also clarified wording in the document.

### CODE REVIEW

File: docs/knowledge-submissions-past-wikipedia.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Status:
 ### Next steps
 
 1. We have to find the correct legal person to find a way to be the correct point person for this project.
-1. Collect suggested places from the community and add them to the above table
+1. Collect [suggested data source domains from the community](https://github.com/instructlab/dev-docs/issues/59) and add them to the above table.
```
Comment (bjhargrave) [inline docs/knowledge-submissions-past-wikipedia.md:45]: This implies the issue will be open "forever" as a receiving site for suggestions. Perhaps a github discussion might be better for this role?

Comment (lhawthorn) [reply docs/knowledge-submissions-past-wikipedia.md:45]: We keep going back and forth on what we plan to use GitHub discussion for vs. issues. I think perhaps the real place to collect this info is in a thread on the community mailing list. Thoughts?

**Review by hickeyma — CHANGES_REQUESTED** [review]
@lhawthorn The current steps in [Process Steps](https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md#process-steps) seem to cover how to handle data source updates. Do you think that this PR as now documented?

**Comment by lhawthorn** [comment]
Yes, I believe this PR is now out of date. I am closing it. Thank you, Martin.

**Pull request closed without merging**